server/datastore/mysql/software_titles_test.go

```diff
@@ -1181,6 +1183,47 @@ func testListSoftwareTitlesAvailableForInstallFilter(t *testing.T, ds *Datastore
 	}, names)
 }
 
+func testListSoftwareTitlesOverflow(t *testing.T, ds *Datastore) {
+	t.Skip("This test is too slow to run in CI")
```

Comment:
Unfortunately running this test on CI breaks the build, as it requires too much resources. Skipping the test, but can be run manually if modifying this code.